src/definition/accessors/IModify.ts

```diff
@@ -554,3 +556,131 @@ export interface IRoomBuilder {
      */
     getRoom(): IRoom;
 }
+
+/**
+ * Interface for creating a user.
+ */
+export interface IUserBuilder {
+  kind: RocketChatAssociationModel.USER;
+
+  /**
+   * Provides a convient way to set the data for the user.
+   * Note: Providing an "id" field here will be ignored.
+   *
+   * @param user the user data to set
+   */
+   setData(user: IUserGenerate): IUserBuilder;
+
+   /**
+    * Sets the email of the user
+    *
+    * @param email the email address of the user
+    */
+    setEmail(email: string): IUserBuilder;
```

Comment:
The email provided here can not exist already in the system otherwise it will cause an error to arise.